src/deadline/job_attachments/asset_manifests/_create_manifest.py

```diff
@@ -38,7 +40,9 @@ def _create_manifest_for_single_root(
             total_input_files=upload_group.total_input_files,
             total_input_bytes=upload_group.total_input_bytes,
             print_function_callback=logger.echo,
-            hashing_progress_callback=hash_callback_manager.callback,
+            hashing_progress_callback=(
```

Comment:
This was a BUG and printed out text for `--json`

Reply:
All unit tests w/ `--json` now does a `json.loads` of the output to make sure json parsing never breaks.